grakn-graql/src/main/java/ai/grakn/graql/internal/query/AbstractQuery.java

```diff
@@ -40,17 +38,7 @@
     @CheckReturnValue
     protected abstract Stream<S> stream();
 
-    @Override
-    public final Stream<String> resultsString(Printer<?> printer) {
-        return results(printer);
-    }
-
-    @Override
-    public final <U> Stream<U> results(GraqlConverter<?, U> converter) {
```

Comment:
why are we getting rid of these?

Reply:
Because we don't want Graql to depend on Printers. It makes no sense for the Graql classes to depend on Printers. Printers should take in Graql Answers. And thus, Printers depend on Graql, not the other way around. The current way causes complexity, which I will explain more in the code review.

Reply:
But converters were part of graql anyway. That way we wouldn't have to have the type checking in `GraqlShell` for instance.